nbs/03_matcher.ipynb

```diff
@@ -345,6 +345,13 @@
     "    # Then filter the list, to contain only the filtered match whose unfiltered version matches the condition\n",
```

Comment:
<pre class="ql-syntax" spellcheck="false">Line #14.        lhs_strs = lhs.split('^')
</pre><p>the new phase symbol should be inside the parser, this is too much of a patch.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/DeanLight/graph_rewrite/pull/17/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='24'/>

Reply:
<p>On it, thanks!</p>